specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2019-05-01/NasCluster.json

```diff
@@ -0,0 +1,207 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Nas Cluster operation endpoints and objects.",
+    "title": "FabricAdminClient",
+    "version": "2019-05-01"
```

Comment:
should it be `2019-05-01-preview`?

Reply:
No. It is the right version

Reply:
but you put the swagger jsons in preview folder.

Reply:
Add bganapa into review. Looks like we put all azure stack hub API into preview folder

Reply:
@MyronFanQiu We have all the the Azurestack Admin APIs in preview folder as we expected breaking changes.  We are mostly stable now. We will plan to move the stable versions under stable folder soon.